src/rebuild/scan.c

```diff
@@ -619,7 +622,45 @@ placement_check(uuid_t co_uuid, vos_iter_entry_t *ent, void *data)
 			if (rc)
 				D_GOTO(out, rc);
 		} else {
-			D_DEBUG(DB_REBUILD, "skip "DF_UOID".\n", DP_UOID(oid));
+			D_DEBUG(DB_REBUILD, "rebuild skip "DF_UOID".\n",
+				DP_UOID(oid));
+			rc = 0;
+		}
+	}
+
+	rebuild_nr = pl_obj_find_reint(map, &md, NULL, rpt->rt_rebuild_ver,
```

Comment:
"rebuild" and "reintegration" should never happen at the same time, but we can't see this  from the function(they are exclusive with each other).